akka-samples/akka-docs-java-lambda/pom.xml

```diff
@@ -27,7 +27,7 @@
     </dependency>
     <dependency>
       <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-stream_2.10</artifactId>
+      <artifactId>akka-stream-experimental_2.10</artifactId>
```

Comment:
is there something I missed why this was not `-experimental`?
When we `publishM2` it publishes as experimental (I moved the publish step in front the pr validation task on jenkins - AFAICS this should work)